### PR TITLE
MAINT-51942: Clean up jitsi started group calls on server startup

### PR DESCRIPTION
**Pull request by hakermi**
**ISSUE**: Some launched group  calls stays on join state for unexpected reasons which leads to a wrong information for the user
**FIX**: Do a clean up of jitsi started group calls

### CODE REVIEW

**Comment by rdenarie** [comment]
@hakermi can you ping me to review the PR ?
As it seems a complex fix, it could be usefull to make pair review, so that you can explain to me your choices

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java [hunk 1]
```diff
@@ -517,7 +519,7 @@ public WebConferencingService(OrganizationService organization,
   protected UserInfo userInfo(String id) throws IdentityStateException {
     User user;
     try {
-      user = organization.getUserHandler().findUserByName(id, UserStatus.ANY);
+      user = organization.getUserHandler().findUserByName(id, UserStatus.ENABLED);
```
Comment (rdenarie) [inline services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: Keep any

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java [hunk 2]
```diff
@@ -1196,6 +1202,30 @@ public CallInfo getCall(String id) throws InvalidCallException {
...
   }
 
+  /**
+   * initialize all saved started calls state.
+   */
+
+  public void initializeStartedCallsState() {
+    try {
+      List<CallEntity> savedCalls = callStorage.findGroupCallsByState(CallState.STARTED);
+      if (!savedCalls.isEmpty()) {
```
Comment (rdenarie) [inline services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: not needed

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java (hunk 1 above)
Comment (hakermi) [reply services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: done

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java (hunk 2 above)
Comment (hakermi) [reply services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: removed

**Review by rdenarie — APPROVED** [review]

**Pull request merged**